data/abilities/credential-access/422526ec-27e9-429a-995b-c686a29561a4.yml

```diff
@@ -10,7 +10,13 @@
   platforms:
     darwin:
       sh:
-        command: cat ~/.bash_history
+        command: find ~/.bash_sessions -name '*' -exec cat {} \; 2>/dev/null
```

Comment:
is there a distinguishable characteristic between an email address and an ssh command? this ability would likely create remote.ssh.cmd facts on both (because email addresses + ssh commands are both likely to be found in bash_history)-- whereas the above ability would not (because only ssh commands would be output from the command)

Reply:
Definitely can add to the parser to search for the term 'ssh' before grabbing anything that has the '@' symbol. Would that cover the edge cases?

Reply:
Yep I think so... if you do that, I think the parser could then be moved into the app/learning/* directory of the core code, where we are putting generic parsers (so you can remove the block from the ability YML file). Parsers in this location will get parsed for every command we get output from.

Reply:
@mchan143 thoughts on moving to learning in core? and taking from here?

Reply:
Outside of the two abilities here (Dump History and Parse SSH Config), I think it's pretty unlikely to see an ssh command in command output and would be wasted overhead in pushing all general command output through this ssh parser in addition to the ip and path parsers. In my opinion, it would be best to leave it in stockpile.